Gemfile

```diff
@@ -3,6 +3,14 @@ source 'https://rubygems.org'
 # Ruby 2.0 is the minimum requirement
 ruby ['2.0.0', RUBY_VERSION].max
 
+# Load vendored dotenv gem and .env file
+gem 'dotenv', :path => File.join(File.dirname(__FILE__), '/vendor/gems/dotenv-2.0.1')
+gem 'dotenv-rails', :path => File.join(File.dirname(__FILE__), '/vendor/gems/dotenv-2.0.1')
+$:.unshift File.join(File.dirname(__FILE__), '/vendor/gems/dotenv-2.0.1/lib')
+require 'dotenv'
+$:.shift
+Dotenv.load(File.join(File.dirname(__FILE__), '.env'))
```

Comment:
I tried to follow the way `dotenv-rails` loads the `.env` files, does that match your intension?